examples/send-msg/main.go

```diff
@@ -52,8 +52,17 @@ func newModel() model {
 	}
 }
 
+func doTick() tea.Cmd {
+	// Simulate activity
+	pause := time.Duration(rand.Int63n(899)+100) * time.Millisecond // nolint:gosec
+	return tea.Tick(pause, func(t time.Time) tea.Msg {
+		// Send a resultMsg to the Update method
```

Comment:
I don't think other models in a single `tea.NewProgram` will receive this message, but if so it might be good to clarify that behavior here, e.g. not just this `Update` method